SpringProject/src/it/java/com/gradproject2019/ConferenceControllerIT.java

```diff
@@ -0,0 +1,198 @@
+package com.gradproject2019;
+
+import com.gradproject2019.conferences.data.ConferenceRequestDto;
+import com.gradproject2019.conferences.data.ConferenceResponseDto;
+import com.gradproject2019.conferences.persistance.Conference;
+import com.gradproject2019.conferences.repository.ConferenceRepository;
+import org.junit.After;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.core.ParameterizedTypeReference;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.test.context.junit4.SpringRunner;
+
+import java.net.URI;
+import java.net.URISyntaxException;
+import java.time.Instant;
+import java.util.List;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment= SpringBootTest.WebEnvironment.RANDOM_PORT)
+public class ConferenceControllerIT {
+
+    @Autowired
+    private ConferenceRepository conferenceRepository;
+
+    @Autowired
+    private TestRestTemplate restTemplate;
+
+    @LocalServerPort
+    int testServerPort = 8080;
+
+    private Conference conference;
+    private String baseUrl;
+    private URI uri;
```

Comment:
BaseUrl & Conference is fine as a field the rest should be local variables instead of fields each test.

Your tests should be isolated from each other, you shouldn't be setting a field in each test.